ai_agents/travel_planner_agent/agent.py

```
@@ -229,7 +229,7 @@ def get_packing_recommendations(
     if trip_days > 5:
         extras.append("Laundry soap sheets or plan for laundry mid-trip")
     if trip_days > 10:
-        extras.append("Extra outfit rotation — pack for 7–10 days and wash")
+        extras.append("Extra outfit rotation - pack for 7–10 days and wash")
```

Ruff (0.15.15)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)